blocks/hero/hero.js

```diff
@@ -13,7 +17,9 @@ const placeholderHtml = '<div class="hero-container" style="visibility: hidden;
 export default async function decorate(block) {
   const id = getBlockId(block);
 
-  const isFirstHero = document.querySelector(".hero.block[data-block-name='hero']").isEqualNode(block);
+  const isFirstHero = document
+    .querySelector('.hero.block[data-block-name="hero"]')
+    .isEqualNode(block);
```

Comment:
This will throw an error on any page without a hero becauee `querySelector` will return `null` which doesn't have the function `isEqualNode`